packages/config/config/devices/0x0063/ge_12730_zw4002.json

```diff
@@ -60,81 +51,6 @@
 					"value": 1
 				}
 			]
-		},
-		"7": {
```

Comment:
I did confirm 7-12 do not work. Here's setting 7 to a non-zero value, but it reads back 0.
```
2021-03-12 20:37:14.280 INFO ZWAVE: Calling api writeValue with args: [ { nodeId: 21, commandClass: 112, property: 7 }, 3, [length]: 2 ]
2021-03-12 20:37:14.281 INFO ZWAVE: Writing 3 to 21-112-0-7
2021-03-12 20:37:14.345 INFO ZWAVE: Success zwave api call writeValue { success: true, message: 'Success zwave api call', result: undefined }
2021-03-12 20:37:15.455 INFO ZWAVE: Node 21: value updated: 112-0-7 3 => 0
```

Reply:
Thanks